pkg/handler/resources.go

```diff
@@ -0,0 +1,136 @@
+package handler
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func GetResources(kubeClient kubernetes.Interface, kind string) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		params := r.URL.Query()
+
+		k := kind
+		if len(k) == 0 {
+			k = strings.ToLower(params.Get("kind"))
+		}
```

Comment:
should you add an 
```go
} else {
    k = strings.ToLower(kind)
}
```
???

Reply:
I started to implement multiple routes as:
```go
	r.HandleFunc("/api/resources", handler.GetResources(kubeClient, ""))
	r.HandleFunc("/api/resources/namespaces", handler.GetResources(kubeClient, "namespace"))
	r.HandleFunc("/api/resources/pods", handler.GetResources(kubeClient, "pod"))
```
passing the `kind` argument to `GetResources` function but at the end I use only `/api/ressources` route passing `kind` as url param. 
What do you prefer ? We can remove other routes if it makes the code clearer

Reply:
Ah, didn't realized that. Let's keep it as you have it now.

Reply:
sure :+1: